update_stage.sh

```diff
@@ -7,4 +7,5 @@ git pull
 
 cd ./bridge_adaptivity
 
-docker-compose -f ./docker-compose-stage.yml restart
+make migrate
```

Comment:
It looks that it should be `migrate-stage`.

Reply:
You are right.